scripts/autotools/libmongoc/CheckSSL.m4

```diff
@@ -144,6 +153,9 @@ AS_IF([test "$PHP_MONGODB_SSL" = "libressl" -o "$PHP_MONGODB_SSL" = "auto"],[
 ])
 
 AS_IF([test "$PHP_MONGODB_SSL" = "auto"],[
+  if test "$crypto_required" = "yes"; then
```

Comment:
Since `$crypto_required` is sometimes unassigned, this is actually a place where you want to do `"x$crypto_required" = "xyes"`.